src/style.css

```diff
@@ -1 +1,100 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  display: flex;
+  justify-content: center; /* Центрування форми по горизонталі */
+  align-items: center;
+  width: 100%; /* Ширина для flex-контейнера */
+  margin-top: 20px;
+}
+
+.input__wrapper {
+  position: relative;
+  width: 100%; /* Задаємо 100% ширину обгортки */
+  display: flex;
+  align-items: center;
+}
+
+.input__icon__1 {
+  position: absolute;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.input__icon__2 {
+  position: absolute;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.input__first {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-color: white;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir-Book, sans-serif;
```

Comment:
You've specified the font-family again for `.input__first` which is redundant since it's already been set for the `body`. If the intention is to have a consistent font throughout the site, setting it on the `body` should suffice.